flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -57,6 +57,11 @@ if [ ! -z "$TF_BUILD" ] ; then
 	on_exit run_on_exit
 fi
 
+if [[ ${PROFILE} == *"enable-declarative-scheduler"* ]]; then
```

Comment:
we also need to disable the kubernetes/yarn e2e tests; forgot to create those tickets.